packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -252,6 +253,9 @@ class DefaultTestVariant extends TestVariant<void> {
   Future<void> tearDown(void value, void memento) async {}
 }
 
+///
+TargetPlatformVariant? defaultPlatformVariant;
```

Comment:
And here. The rest of the changes are just a whole lot of opt-outs for the broken tests